packages/algob/src/lib/compile.ts

```diff
@@ -31,9 +31,13 @@ export class CompileOp {
   //   (Examples: `mysc.teal, security/rbac.teal`)
   //   MUST have a .teal, .lsig or .py extension
   // @param force: if true it will force recompilation even if the cache is up to date.
-  async ensureCompiled (filename: string, force: boolean): Promise<ASCCache> {
+  async ensureCompiled (filename: string, force?: boolean, scInitParam?: Object): Promise<ASCCache> {
```

Comment:
Please update the docstring to explain parameters.

Reply:
ok